configs/from_http.json

```diff
@@ -58,7 +58,9 @@
     "max_retries": 5,
     "model_max_batch_size": 64,
     "pipeline_batch_size": 1024,
-    "use_uvloop": true
+    "use_uvloop": true,
+    "use_dependency_tool": false,
+    "assume_vulnerable_package_present": true
```

Comment:
I think you should assume this always when the `use_dependency_tool` is `False` and never when `True`. Is there any other use case I don't see?

Reply:
No, just a future thinking for extension point.
as agreed, in the meantime, we will not use it.